@xen-orchestra/web-core/lib/assets/css/_colors.pcss

```diff
@@ -0,0 +1,121 @@
+:root {
+  --color-grey-000: #000000;
+  --color-grey-100: #1a1b38;
+  --color-grey-200: #595a6f;
+  --color-grey-300: #9899a5;
+  --color-grey-400: #bfbfc6;
+  --color-grey-500: #e5e5e7;
+  --color-grey-600: #ffffff;
+
+  --background-color-primary: #ffffff;
+  --background-color-secondary: #f6f6f7;
+
+  --color-purple-base: #8f84ff;
+  --color-purple-d20: color(#8f84ff blend(black 20%));
+  --color-purple-d40: color(#8f84ff blend(black 40%));
+  --color-purple-d60: color(#8f84ff blend(black 60%));
+  --color-purple-l20: color(#8f84ff blend(white 20%));
+  --color-purple-l40: color(#8f84ff blend(white 40%));
+  --color-purple-l60: color(#8f84ff blend(white 60%));
+  --background-color-purple-10: color(white blend(#8f84ff 10%));
+  --background-color-purple-20: color(white blend(#8f84ff 20%));
+  --background-color-purple-30: color(white blend(#8f84ff 30%));
+  --background-color-purple-60: color(white blend(#8f84ff 60%));
+
+  --color-green-base: #2ca878;
+  --color-green-d20: color(#2ca878 blend(black 20%));
+  --color-green-d40: color(#2ca878 blend(black 40%));
+  --color-green-d60: color(#2ca878 blend(black 60%));
+  --color-green-l20: color(#2ca878 blend(white 20%));
+  --color-green-l40: color(#2ca878 blend(white 40%));
+  --color-green-l60: color(#2ca878 blend(white 60%));
+  --background-color-green-10: color(white blend(#2ca878 10%));
+  --background-color-green-20: color(white blend(#2ca878 20%));
+  --background-color-green-30: color(white blend(#2ca878 30%));
+  --background-color-green-60: color(white blend(#2ca878 60%));
+
+  --color-orange-base: #ef7f18;
+  --color-orange-d20: color(#ef7f18 blend(black 20%));
+  --color-orange-d40: color(#ef7f18 blend(black 40%));
+  --color-orange-d60: color(#ef7f18 blend(black 60%));
+  --color-orange-l20: color(#ef7f18 blend(white 20%));
+  --color-orange-l40: color(#ef7f18 blend(white 40%));
+  --color-orange-l60: color(#ef7f18 blend(white 60%));
+  --background-color-orange-10: color(white blend(#ef7f18 10%));
+  --background-color-orange-20: color(white blend(#ef7f18 20%));
+  --background-color-orange-30: color(white blend(#ef7f18 30%));
+  --background-color-orange-60: color(white blend(#ef7f18 60%));
+
+  --color-red-base: #be1621;
+  --color-red-d20: color(#be1621 blend(black 20%));
+  --color-red-d40: color(#be1621 blend(black 40%));
+  --color-red-d60: color(#be1621 blend(black 60%));
+  --color-red-l20: color(#be1621 blend(white 20%));
+  --color-red-l40: color(#be1621 blend(white 40%));
+  --color-red-l60: color(#be1621 blend(white 60%));
+  --background-color-red-10: color(white blend(#be1621 10%));
+  --background-color-red-20: color(white blend(#be1621 20%));
+  --background-color-red-30: color(white blend(#be1621 30%));
+  --background-color-red-60: color(white blend(#be1621 60%));
+}
+
+:root.dark {
+  --color-grey-000: #ffffff;
+  --color-grey-100: #e5e5e7;
+  --color-grey-400: #595a6f;
+  --color-grey-200: #bfbfc6;
+  --color-grey-300: #9899a5;
+  --color-grey-500: #1a1b38;
```

Comment:
In the design system, the value is `#3a3b54` for `grey-500`

Reply:
Good catch 👍🏻